curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/replace-loops-using-recursion.english.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 5cfa3679138e7d9595b9d9d4
+title: Replace Loops using Recursion
+challengeType: 1
+videoUrl: 'https://www.freecodecamp.org/news/how-recursion-works-explained-with-flowcharts-and-a-video-de61f40cb7f9/'
+---
+
+## Description
+<section id='description'>
+Recursion is the concept that a solution can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the first <code>n</code> elements in an array to create the product of the elements. Using a <code>for</code> loop, you could do this:
```

Comment:
```suggestion
Recursion is the concept that a function can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the first <code>n</code> elements in an array to create the product of the elements. Using a <code>for</code> loop, you could do this:
```
I feel like using 'function' instead of 'solution' will help making things clearer.

Reply:
Agreed, thanks!